xsensmt/XsensMT.cpp

```diff
@@ -364,15 +364,19 @@ bool XsensMT::open(yarp::os::Searchable &config)
         return false;
     }
 
+    // reset the orientation if the 'reset' parameter is true
     if(config.check("reset"))
     {
-        bool okReset = m_xsensDevice->resetOrientation(XRM_Alignment);
-        if(okReset)
+        if(config.find("reset").asBool())
```

Comment:
I think it do the reset if it finds `reset`, whether it's true or false!

Reply:
I just changed this part and it checks if it true or false.